test/e2e/wasm/authz/authz_bench_integration_test.go

```diff
@@ -120,9 +120,9 @@ func BenchmarkRESTAuthzAllow10Paths(b *testing.B) {
 }
 
 // TODO: Re-enable when performance issues have been addressed.
```

Comment:
[nit] Let's move the comment to the commented-out block below. We _could_ also try enabling that again, I suppose? 🤔

Reply:
Oops.  Yep, will do.  Will give the 1000Paths a try when I merge all the squash the two commits and add some tests for bulk free.  If it seems table I can re-enable it...